### PR TITLE
Italian translation

### PR DESCRIPTION
**Pull request by antoniobonanno**
Created Italian translation for solidus_virtual_gift_card

### CODE REVIEW

**Comment by bosskovic** [comment]
Is there something I can do to help this PR get accepted and merged into the master, so that we can switch to using the main gem instead of a fork?

File: solidus_virtual_gift_card.gemspec [hunk 1]
```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.require_path = "lib"
   s.requirements << "none"
 
-  s.add_dependency "solidus", "~> 1.2"
+  s.add_dependency "solidus_core", "~> 1.2"
```
Comment (gmacdougall) [inline solidus_virtual_gift_card.gemspec]: This is not a safe dependency change. There are also dependencies on the API and backend in controller inheritance.

Comment (bosskovic) [inline solidus_virtual_gift_card.gemspec]: is it enough if I just add dependencies to backend and api?

**Comment by gmacdougall** [comment]
We currently have a failing master build. If you wanted to take a quick look at that, it would expedite things. We will want to see a green build before merging.

File: solidus_virtual_gift_card.gemspec (hunk 1 above)
Comment (gmacdougall) [inline solidus_virtual_gift_card.gemspec]: I believe so, I haven't verified that those are the only other dependencies, but a quick glance showed that to be the case.

**Comment by adammathys** [comment]
Merged the translations in #37.

**Pull request closed without merging**